x-pack/plugins/security_solution/public/common/components/events_tab/events_query_tab_body.tsx

```diff
@@ -84,7 +86,11 @@ const EventsQueryTabBodyComponent: React.FC<EventsQueryTabBodyComponentProps> =
 }) => {
   const dispatch = useDispatch();
   const { globalFullScreen } = useGlobalFullScreen();
-  const ACTION_BUTTON_COUNT = 4;
+  const { uiSettings } = useKibana().services;
+
+  const isSessionViewEnabled = uiSettings.get(ENABLE_SESSION_VIEW_PLUGIN);
+
+  const ACTION_BUTTON_COUNT = isSessionViewEnabled ? 5 : 4;
```

Comment:
We might add the unit test later(follow up PR?) for checking the buttons count on dependency if the session view is enabled

Reply:
ya no problem, once someone confirms if we are keeping the flag or not 😆

Reply:
hey @kqualters-elastic, we can drop the Feature Flag

cc @qcorporation

Reply:
Thanks, @opauloh for cc.ing me. I can confirm we want to keep the empty state.
I remember we had originally talked about having a Kibana flag that would turn off Session View within the initial integration discussions but there's value in having an empty state.

visibility: @norrietaylor @snehsach19